lib/bandit/http1/adapter.ex

```diff
@@ -109,6 +112,21 @@ defmodule Bandit.HTTP1.Adapter do
   defp get_version({1, 0}), do: {:ok, :"HTTP/1.0"}
   defp get_version(other), do: {:error, "invalid HTTP version: #{inspect(other)}"}
 
+  defp get_content_length(headers) do
+    with {_, value} <- List.keyfind(headers, "content-length", 0),
+         [length] <- Enum.uniq(Plug.Conn.Utils.list(value)),
```

Comment:
Related text in the [RFC9112](https://www.rfc-editor.org/rfc/rfc9112.html#section-6.3)
> If a message is received without [Transfer-Encoding](https://www.rfc-editor.org/rfc/rfc9112.html#field.transfer-encoding) and with an invalid [Content-Length](https://www.rfc-editor.org/rfc/rfc9112.html#body.content-length) header field, then the message framing is invalid and the recipient MUST treat it as an unrecoverable error, unless the field value can be successfully parsed as a comma-separated list ([Section 5.6.1](https://www.rfc-editor.org/rfc/rfc9110#section-5.6.1) of [[HTTP](https://www.rfc-editor.org/rfc/rfc9112.html#HTTP)]), all values in the list are valid, and all values in the list are the same (in which case, the message is processed with that single value used as the Content-Length field value)